src/mime/custom_mime_raw

```diff
@@ -0,0 +1,4 @@
+# RAW data from flash
+
+8	string	\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff	RAW
+!:mime data/raw
```

Comment:
seems new line is missing here which leeds to warning 
```
custommime, 4361: Warning: MIME type ` data/raw# ====================== UEFI ======================' has bad char '#'
```
when compiling the file